services/tenant-ui/frontend/src/components/issuance/Schemas.vue

```diff
@@ -3,78 +3,47 @@
 
   <ProgressSpinner v-if="loading" />
   <div v-else>
-    <DataTable
```

Comment:
i think we like the formatter breaking these options into separate lines?

Reply:
Yeah I like it as well, looking into how we can get auto-formatter to pick up the settings properly though...

Reply:
Tracking ongoing formatter needs in https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-401 can hopefully address next sprint ASAP